docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -186,6 +193,8 @@
 
     // Targeting only the first level of navigation on desktop
     .hb-main-nav__menu-lv1 > li > & {
+      @include hb-global-color('color', 'black');
```

Comment:
does this need to be gray-dark for traditional?

Reply:
Nope. :smile: I checked the design comps and the very top level of the nav remains black.

![top level](https://user-images.githubusercontent.com/12678977/82101358-50272100-96da-11ea-96d5-cb1189185f73.png)